examples/docker-build/dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:10-stretch-slim as builder
+
+WORKDIR /src
+
+# Remove unnecessary files from node_modules (.md, .ts, ...)
+RUN cd /tmp && \
+  curl -#L https://github.com/tj/node-prune/releases/download/v1.0.1/node-prune_1.0.1_linux_amd64.tar.gz | tar -xvzf- && \
```

Comment:
node-prune, has known (tj/node-prune#50) issues that may introduce problems for users. `.yarnclean` is more adviced.

Reply:
`.yarnclean`:

```
*.yaml
*.md
*.lock
LICENSE
CHANGELOG
```